src/templates/find_usages.html

```diff
@@ -1,4 +1,4 @@
-<table class="table table-condensed">
+<table class="table table-hover">
```

Comment:
I forgot the table will be a lot smaller because it's question-specific now. Yeah, leave as is, sure.